administrator-manual/en/webtop5.rst

```diff
@@ -185,6 +185,75 @@ To do so, just right click on the shared resource → Customize → Sync. device
 
 The default setting is “Not active”.
 
+Synchronization with CalDAV and CardDAV
+=======================================
+
+Calendars and address books can be synchronized also through :index:`CalDAV and CardDAV protocols`.
+
+To synchronize a calendar, pick up its ``URL`` link right-clicking on the calendar and selecting :guilabel:`Links to this calendar`,
+then use it to configure your third-party client.
+
+To synchronize an address book, pick up its ``URL`` link right-clicking on the address book and selecting :guilabel:`Links to this addressbook`,
+then use it to configure your third-party client.
+
+For the authentication provide your credentials in the following form:
+
+- **User name**: enter your full user name (i.e. *goofy@nethserver.org*)
+- **Password**: enter your password
+
+Some third-party clients allow to simplify the configuration through the *autodiscovery* feature that automatically discovers the 
+synchronizable resources, as in the case of mobile devices clients (i.e. Android or iOS devices).
+
+
+.. note::
+
+   If you are using clients that do not support autodiscovery, you need to use the full URL: ``https://<server_name>/webtop-dav/server.php``
+
+
+Google Android
+--------------
+
+- install the `Opensync <https://deependhulla.com/android-apps/opensync-app>`_ application from the market;
+- add a new account clicking on "``+``" key and select :guilabel:`Login with URL and username` method;
+- insert the ``URL`` (``https://<server_name>``), complete username (i.e. *goofy@nethserver.org*) and password;
+- click on the new profile and select the resources you want to synchronize.
+
+Apple iOS
+---------
+
+- Settings -> Account and Password -> Add account;
+- select :guilabel:`Other` -> Add :guilabel:`CalDAV` or :guilabel:`CardDAV` account;
+- insert the server name (i.e. *server.nethserver.org*), complete username (i.e. *goofy@nethserver.org*) and password.
+
+.. note::
+
+   At the moment mobile devices support **only personal resources synchronization**.
+ 
+By default the syncronization ``URL`` uses the server principal name (``FQDN``), if you need to change it: ::
+
+ config setprop webtop DavServerUrl https://<new_name_server>/webtop-dav/server.php
+ signal-event nethserver-webtop5-update
+
+
+Tips for some desktop clients
```

Comment:
Change to
"Desktop clients"

Add an introduction for desktop client, for example that you need to install CardBook and Lightning in TB.